tests/attestation_plugin/plugin/tests.c

```diff
@@ -571,6 +610,36 @@ static void _test_claims(
                                      sgx_endorsements->items[j].data,
                                      sgx_endorsements->items[j].size) == 0);
         }
+
+        if (!is_local)
+        {
+            oe_parsed_tcb_info_t parsed_tcb_info;
+            uint8_t fmspc_size = sizeof(parsed_tcb_info.fmspc);
+            uint8_t* fmspc = (uint8_t*)calloc(fmspc_size, sizeof(uint8_t));
+            int i;
+
+            _find_fmspc(
+                sgx_endorsements->items[OE_SGX_ENDORSEMENT_FIELD_TCB_INFO].data,
+                fmspc,
+                fmspc_size);
+            uint8_t* p_value =
+                (uint8_t*)_find_claim(claims, claims_size, OE_CLAIM_SGX_FMSPC);
+            for (i = 0; i < fmspc_size; i++)
+            {
+                printf(
+                    "value[%d]: %hhu, fmspc[%d]: %hhu\n",
```

Comment:
the logs say it is not enterering this loop. Can you print out the value of fmspc_size somewhere?